templates/cra-template-brightsign-app/template/src/autorun.brs

```diff
@@ -8,13 +8,13 @@ function main()
   ' Enable SSH
   enableSSH()
 
+  ' Initialize roNodeJs with path or correct filename, whether webpack is used or not.
+  node_js = CreateObject("roNodeJs", "sd:/dist/backend.js", {message_port: mp, node_arguments: ["--inspect=0.0.0.0:2999"], arguments: []})
```

Comment:
Is moving this line before creating the HTML widget creation, part of the solution?

Reply:
While it does not directly affect the issue's reported error of impacting the chrome inspect, it enables the html to be seen on the display when in this order. Before, the screen was black.